src/index.js

```diff
@@ -59,7 +59,10 @@ function loginUser() {
   $('.login-form').submit(e => {
     e.preventDefault();
   })
-  console.log(tripData)
+  let arrayOfFifty = []
```

Comment:
Just curious:
What's this doing? I couldn't figure it out!

Reply:
So essentially I am generating an array of 50 items. traveler1 -traveler50.

This allows me to check the input value of the username and have something to compare it to. THen I split the string value and just splice out the number part so i can instantiate a new instance of my traveller class